build-system/tasks/coverage-map/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "private": true,
+  "name": "gulp-coverage-map",
+  "version": "0.1.0",
+  "description": "Gulp coverage map.",
+  "dependencies": {
+    "puppeteer": "5.2.1",
```

Comment:
At the moment, Travis is failing. I wonder if it has to do with this.
@rsimha @dvoytenko